website/docs/integrations/intellij.mdx

```diff
@@ -67,7 +67,7 @@ The plugin is compatible with the following IDEs. Check the plugin [JetBrains Ma
 
 ### Authentication
 
-1. Get your ConfigCat Public API credentials from [ConfigCat Dashboard/Public Management API credentials](https://app.configcat.com/my-account/public-api-credentials).
+1. Get your ConfigCat Public API credentials from [ConfigCat Public Management API](https://app.configcat.com/my-account/public-api-credentials).
```

Comment:
Sorry, I just noticed this minor problem.

You get the credentials *on* the Dashboard, *for* accessing the API. There is some repeating of words we should reduce too.

```suggestion
1. Get your API credentials for [ConfigCat Public Management API](https://app.configcat.com/my-account/public-api-credentials).
```

Could you fix all occurrences of this? Last request, I promise :)